util/install.sh

```diff
@@ -226,6 +286,11 @@ install_etcd() {
     create_project_dir $project_prefix
     copy_file "$dst/etcd" "$project_prefix/sbin"
     copy_file "$dst/etcdctl" "$project_prefix/sbin"
+    gen_servicectl \
```

Comment:
ditto. Each `install_xx` judges whether the execution is successful, and terminates the script if it fails.

Reply:
This feature is already included in `gen_servicectl`.